### DISCUSSION TITLE
Languages and Algorithms for Artificial Intellingence

### DISCUSSION THREAD

**Discussion by Danysan1** [montali/unibo-ai#4] [General]
Thread to discuss about LAAI.

The material for each module can be found in:

1. Gabbrielli -> [link](https://github.com/montali/unibo-ai/tree/master/Languages%20and%20algorithms%20for%20AI%20-%20Module%201)
2. Zavattaro -> [link](https://github.com/montali/unibo-ai/tree/master/Languages%20and%20algorithms%20for%20AI%20-%20Module%202)
3. Dal Lago -> TBD

**Comment by Danysan1** [comment]
@montali I've uploaded today's examples [here](https://github.com/montali/unibo-ai/tree/master/Languages%20and%20algorithms%20for%20AI%20-%20Module%202/examples) and linked them inside [1 - Introduction.md](https://github.com/montali/unibo-ai/blob/master/Languages%20and%20algorithms%20for%20AI%20-%20Module%202/1%20-%20Introduction.md)

**Reply by montali** [reply]
Great! Sorry about the scarcity of my notes for LAAI but I really don't like that lol, if you have anything to add feel free to do so! Grazie come sempre✨
